js/main.js

```diff
@@ -111,47 +122,83 @@ function putLyrics(lyrics) {
+    lyricLi.append(lyricP1, lyricP2, lyricP3);
+    lyricP1.setAttribute('lyric-line', count);
+    lyricP1.textContent = lyricsArray[count];
+    count++;
+    lyricP2.setAttribute('lyric-line', count);
+    lyricP2.textContent = lyricsArray[count];
+    count++;
+    lyricP3.setAttribute('lyric-line', count);
+    lyricP3.textContent = lyricsArray[count];
+    count++;
+    cardCount++;
+    data.lyrics.push([lyricsArray[i], lyricsArray[i + 1], lyricsArray[i + 2]]);
+    $divCardLyrics.append(lyricLi);
     data.totalLyricCards++;
   }
   for (var j = 0; j < cardCount; j++) {
     randomizeMissingLyrics(j.toString());
   }
+  $cardIndicator.textContent = data.lyricCard + '/' + (data.totalLyricCards - 1);
 }
 
 function randomizeMissingLyrics(lyricCardNumber) {
   var $cardLyrics = document.querySelectorAll('.lyrics');
   var $p = $cardLyrics[lyricCardNumber].querySelectorAll('p');
   var randomLine = Math.floor(Math.random() * $p.length);
+  data.randomLyricLine.push(randomLine);
   var wordsOfRandomLine = $p[randomLine].textContent.split(' ');
-  var randomIndex = Math.floor(Math.random() * wordsOfRandomLine.length);
+  var randomIndex = Math.floor(Math.random() * (wordsOfRandomLine.length - 1));
   var randomActualWords = '';
-  randomActualWords = wordsOfRandomLine[randomIndex] + ' ' + wordsOfRandomLine[randomIndex + 1];
+  randomActualWords = [wordsOfRandomLine[randomIndex], wordsOfRandomLine[randomIndex + 1]];
+  var splitted = $p[randomLine].textContent.split(randomActualWords[0] + ' ' + randomActualWords[1]);
   data.missingWords.push(randomActualWords);
-  wordsOfRandomLine.splice(randomIndex, 2, '_______');
-  $p[randomLine].textContent = wordsOfRandomLine.join(' ');
+  var $span1 = document.createElement('span');
+  var $span2 = document.createElement('span');
+  $span1.textContent = '___';
+  $span1.className = 'span1';
+  $span2.textContent = '___';
+  $span2.className = 'span2';
+  $p[randomLine].textContent = '';
+  $p[randomLine].append(splitted[0], $span1, $span2, splitted[1]);
+}
+
+function handleSubmit(event) {
+  var $cardLyrics = document.querySelectorAll('.lyrics');
+  var $p = $cardLyrics[data.lyricCard].querySelectorAll('p');
+  var $span = $p[data.randomLyricLine[data.lyricCard]].querySelectorAll('span');
+
+  data.submittedWords.push($lyricsInput.value);
+  var $wordsOfInput = $lyricsInput.value.split(' ');
+  if ($wordsOfInput[0] === data.missingWords[data.lyricCard][0]) {
+    $span[0].className = 'correct';
+    $span[0].textContent = data.missingWords[data.lyricCard][0] + ' ';
+  } else {
+    $span[0].className = 'incorrect';
+    $span[0].textContent = data.missingWords[data.lyricCard][0] + ' ';
+  }
+  if ($wordsOfInput[1] === data.missingWords[data.lyricCard][1]) {
+    $span[1].className = 'correct';
+    $span[1].textContent = data.missingWords[data.lyricCard][1];
+  } else {
+    $span[1].className = 'incorrect';
+    $span[1].textContent = data.missingWords[data.lyricCard][1];
+  }
```

Comment:
You should just do this with a loop. If you ever want to increase the number of words you are dealing with, you will need to add additional conditionals for each word with this setup. If you use a loop, you can use the same code no matter how many words there are.